step_implementations/specs/configuration_spec.rb

```diff
@@ -97,21 +97,6 @@
   basic_configuration.change_cruise_config_file_to 'invalid-cruise-config.xml'
 end
 
-step 'Changing the artifacts location to <artifact_location>' do |artifact_location|
-  basic_configuration.set_artifact_location artifact_location
-end 
```

Comment:
Any reason why these steps are deleted?

Reply:
Actually, when I pulled upstream master, then the those step were duplicated. so I removed those steps